src/containers/profiles/state/reducers.test.js

```diff
@@ -0,0 +1,73 @@
+import reducer from './reducers';
+import { profile, state, errors } from './mock';
+import {
+  USER_PROFILE_ERROR, EDIT_PROFILE_ERROR, EDIT_USER_PROFILE, VIEW_USER_PROFILE,
+} from './types';
```

Comment:
Consider import each of this in a new line because they are more than  three imports